resume.md

```diff
@@ -1,49 +1,51 @@
-# Alex Takahashi
+# RAHIL SHAH
 
-(123) 456 - 7890 | email@gmail.com
+(647) 972 - 0150 | rahil190@gmail.com
+
+rahil.me | linkedin.com/in/rahilllshah
 
-alextakahashi.github.io | github.com/alextakahashi | inkedin.com/in/alextakahashi/
 
 ## EDUCATION
-**University of California Berkeley** | B.S. Bioengineering, Computer Science Minor ... *May 2017*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**McMaster University** | BS Computer Science ... *Jun '22*
+
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+* **Design & Prototyping**: Adobe CC Software, Figma,  InVision, Sketch, Visual design, interaction design, responsive design, prototyping, user-centered design, usability testing
+* **Development**: HTML, CSS, JavaScript, Java, C, Python, SQL, Git
+* **Soft Skills**: Communication, organization, leadership, problem-solver, strong work ethic, responsible
+
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Product Designer**: CanTrace | *Apr '20 - Aug '20*
+* Designed and presented a distinguished app integrating healthcare and contact tracing technology to the provincial and federal government of Canada
+* Designed flows, developed product strategy and continually solicited feedback to improve app
+
+**Front-End Developer**: Schulich School of Medicine & Dentistry | *Jan '19 - Present*
+* Work with the nephrology division teaching faculty to develop several facets of their websites and applications which has led to the recruitment of many highly qualified personnel
+
+**Web Design Intern**: Nova Solutions | *May '19 - Aug '19*
+* Designed and developed mockups, web pages for 3-5 clients per week using Adobe CC Software and web development tools totaling 30+ projects
+* Worked with company executives on high-level strategic decisions that increased company website engagement by 27%
```

Comment:
how did you increase engagement?  What does engagement mean?  is this click through rate?  How did you measure this?